src/plugins/index.js

```diff
@@ -1,8 +1,13 @@
 const pino = require('./hapi-pino.js');
-
+const swagger = require('./hapi-swagger.js');
+const Inert = require('@hapi/inert');
```

Comment:
Make naming schema constant. pino and swagger starts with small letters whereas inert and vision starts with Capital letters